### PR TITLE
(FOROME-1196) Updated "list-value-attr-filter" test cases 

### PR DESCRIPTION
**Pull request by Mariamkikalia**
https://quantori.atlassian.net/browse/FOROME-1196

### CODE REVIEW

**Review by Gjeriashvili — APPROVED** [review]

File: qa/tests/features/filter-refiner/list-value-attr-filter.feature [hunk 1]
```diff
@@ -2,25 +2,25 @@ Feature: Filter Refiner, Filter by List-value attributes
...
-		Then Filter Refiner should be opened
+		Then the "Filter Refiner" should be opened
 
-	Scenario Outline: Search by attribute value: full value
-		Given the Filter Refiner for the "xl_PGP3140_wgs_NIST-4_2" was open
-		When the user clicks the "ClinVar_Significance" attribute
-		And attribute's values are displayed in the middle of the screen
-		And the user enters the <Attribute Value> in the Search field
+	Scenario Outline: 03 Search by attribute value: full value
+		Given the "Filter Refiner" for the "xl_PGP3140_wgs_NIST-4_2" was open
```
Comment (mbredeleva) [inline qa/tests/features/filter-refiner/list-value-attr-filter.feature]: was open -> was opened

File: qa/tests/features/filter-refiner/list-value-attr-filter.feature [hunk 2]
```diff
@@ -29,11 +29,11 @@ Feature: Filter Refiner, Filter by List-value attributes
 		| drug response       |
 		| protective          |
 
-	Scenario Outline: Search by attribute value: substring
-		Given the Filter Refiner for the "xl_PGP3140_wgs_NIST-4_2" was open
-		When the user clicks the "Chemicals" attribute
-		And attribute's values are displayed in the middle of the screen
-		And the user enters the <Substr Value> in the Search field
+	Scenario Outline: 04 Search by attribute value: substring
+		Given the "Filter Refiner" for the "xl_PGP3140_wgs_NIST-4_2" was open
```
Comment (mbredeleva) [inline qa/tests/features/filter-refiner/list-value-attr-filter.feature]: was open -> was opened

File: qa/tests/features/filter-refiner/list-value-attr-filter.feature [hunk 3]
```diff
@@ -43,11 +43,11 @@ Feature: Filter Refiner, Filter by List-value attributes
 		| prednisolone            | isolo          |
 		| zoledronate             | edron          |
 
-	Scenario Outline: Search by attribute value: upper-case
-		Given the Filter Refiner for the "xl_PGP3140_wgs_NIST-4_2" was open
-		When the user clicks the "ClinVar_Significance_Invitae" attribute
-		And attribute's values are displayed in the middle of the screen
-		And the user enters the <Upper Value> in the Search field
+	Scenario Outline: 05 Search by attribute value: upper-case
+		Given the "Filter Refiner" for the "xl_PGP3140_wgs_NIST-4_2" was open
```
Comment (mbredeleva) [inline qa/tests/features/filter-refiner/list-value-attr-filter.feature]: the same ))

File: qa/tests/features/filter-refiner/list-value-attr-filter.feature [hunk 4]
```diff
@@ -56,29 +56,30 @@ Feature: Filter Refiner, Filter by List-value attributes
 		| pathogenic        | PATHOGENIC     |
 
 
-	Scenario: Search by attribute value: lower-case
-		Given the Filter Refiner for the "xl_PGP3140_wgs_NIST-4_2" was open
-		When the user clicks the "Callers" attribute
-		And attribute's values are displayed in the middle of the screen
-		And the user enters the <Lower Value> in the Search field
+	Scenario Outline: 06 Search by attribute value: lower-case
+		Given the "Filter Refiner" for the "xl_PGP3140_wgs_NIST-4_2" was open
```
Comment (mbredeleva) [inline qa/tests/features/filter-refiner/list-value-attr-filter.feature]: the same here and below

**Review by mbredeleva — CHANGES_REQUESTED** [review]

File: qa/tests/features/filter-refiner/list-value-attr-filter.feature (hunk 1 above)
Comment (Mariamkikalia) [reply qa/tests/features/filter-refiner/list-value-attr-filter.feature]: Done

File: qa/tests/features/filter-refiner/list-value-attr-filter.feature (hunk 2 above)
Comment (Mariamkikalia) [reply qa/tests/features/filter-refiner/list-value-attr-filter.feature]: Done

File: qa/tests/features/filter-refiner/list-value-attr-filter.feature (hunk 3 above)
Comment (Mariamkikalia) [reply qa/tests/features/filter-refiner/list-value-attr-filter.feature]: Done

File: qa/tests/features/filter-refiner/list-value-attr-filter.feature (hunk 4 above)
Comment (Mariamkikalia) [reply qa/tests/features/filter-refiner/list-value-attr-filter.feature]: Done

**Review by mbredeleva — APPROVED** [review]

**Pull request merged**